python/ray/workflow/workflow_state_from_dag.py

```diff
@@ -168,7 +168,10 @@ def _node_visitor(node: Any) -> Any:
                     flattened_args = _SerializationContextPreservingWrapper(
                         flattened_args
                     )
-                input_placeholder: ray.ObjectRef = ray.put(flattened_args)
+                workflow_manager = workflow_access.get_management_actor()
```

Comment:
we should get the workflow actor manager at the beginning of the function instead, so we won't get the actors repeatedly for every object